src/AddTodoForm.js

```diff
@@ -1,24 +1,28 @@
-import React from "react"
+import React, { useState } from "react"
 
+function AddTodoForm(AddTodoForm) {
+  const [todoTitle, setTodoTitle] = useState()
 
-const todoTitle 
+function handleTitleChange(event){
+  let newTodoTitle=event.target.value;
+  console.log(event.target.value)
+  setTodoTitle(newTodoTitle);
 
-function AddTodoForm(props) {
 
-    function handleAddTodo(event) {
-            event.preventDefault()
-            let todoTitle = event.target.title.value
-            event.target.title.value = ""
-            console.log(todoTitle)
-            props.onAddTodo(todoTitle)
-    }
-    return (
-        <form onSubmit={handleAddTodo}>
-        <label htmlFor="todoTitle"> Title</label>
-        <input type="text" id="todoTitle" name="title" />
-        <button type="submit">Add</button>
-        </form>
-    )
 }
+  function handleAddTodo(event) {
+    event.preventDefault()
 
-export default AddTodoForm
\ No newline at end of file
+    // let todoTitle = event.target.title.value
+    // event.target.title.value = ""
+    console.log(todoTitle)
+    AddTodoForm.onAddTodo({title:todoTitle,id:Date.now()})  
```

Comment:
this doesn 't make sense, you want props.onAddTodo({title:todoTitle,id:Date.now()})